frontend/docker-compose-e2e-tests.yml

```diff
@@ -43,7 +58,9 @@ services:
     ports:
       - 8080:8080
     depends_on:
-      - flagsmith-api
+      flagsmith-api:
+        condition: service_started
```

Comment:
Does this offer anything additional compared to how it was before?

Reply:
Right, no difference is documented. I changed `service_started` with `service_healthy` and added `healthcheck` to `flagsmith-api` so we can test this locally. At least in my environments runs with this modification.